packages/pg/test/test-helper.js

```diff
@@ -111,16 +111,6 @@ assert.success = function (callback) {
   }
 }
 
-assert.throws = function (offender) {
-  try {
-    offender()
-  } catch (e) {
-    assert.ok(e instanceof Error, 'Expected ' + offender + ' to throw instances of Error')
-    return
-  }
-  assert.ok(false, 'Expected ' + offender + ' to throw exception')
-}
-
```

Comment:
Yeah, good call.  Monkey-patching assert was such a bad idea I had such a long time ago.  The use of globals throughout some of the tests is also gross....need to fix up one day soonish. 🤦